setup/install/providers/ecs.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: single
+title:  "ECS (Amazon Web Services)"
+sidebar:
+  nav: setup
+redirect_from: /setup/providers/ecs/
+---
+
+{% include toc %}
+
+In the ECS cloud provider, an [__Account__](/concepts/providers/#accounts)
+maps to a Spinnaker AWS account, which itself is able to authenticate against a given [AWS
+account](https://aws.amazon.com/account/).
+
+## Prerequisites
+
+### ECS Cluster
+You need to [create an ECS cluster](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/create_cluster.html) and have enough capacity in it to deploy your containers.
+Please note that ECS clusters are not the same concept as [Spinnaker clusters](/concepts/clusters/).  
```

Comment:
@BrunoCarrier is it worth elaborating on the difference here?

Reply:
Since the explanation is part of the glossary, I will remove this part as per your suggestion